taxcalc/behavior.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+class Behavior(object): 
+
+	def __init__(self, calcX, calcY, elast_wrt_atr = 0.4, inc_effect = 0.15):
+		self.calcX = calcX
+		self.calcY = calcY
+		self.elast_wrt_atr = elast_wrt_atr
+		self.inc_effect = inc_effect
+
+	@property
+	def calc(self):
```

Comment:
Since `calc` mutates the state of a `Behavior` object (well, actually its constituent calculators), I think it makes more sense to have this as a function, instead of a property.